packages/core/state/selection/actions.ts

```diff
@@ -75,6 +78,187 @@ export function removeFileFilter(filter: FileFilter | FileFilter[]): RemoveFileF
     };
 }
 
+/**
+ * SET_FUZZY_FILTERS
+ *
+ * Intention to set, wholesale, a list of FuzzyFilters into application state. This should not be dispatched
```

Comment:
what does wholesale mean here?

Reply:
copy pasta comment from a different filter type. I believe intended to mean "everything at once instead of individual changes"